ModInstances/SpookyNPC.cs

```diff
@@ -15,128 +15,93 @@
 
 namespace SpookyTerraria
 {
-	public class SpookyNPCs : GlobalNPC
+    public class SpookyNPCs : GlobalNPC
     {
         public override bool InstancePerEntity => true;
         public override bool CloneNewInstances => true;
 
         public int heartBeatTimer;
+
         public override void SetupShop(int type, Chest shop, ref int nextSlot)
         {
             switch (type)
             {
                 case NPCID.Merchant:
                     shop.item[nextSlot].SetDefaults(ModContent.ItemType<Battery>());
-                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(platinum: 0, gold: 0, silver: 10, copper: 0);
+                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(0, 0, 10, 0);
                     nextSlot++;
                     break;
                 case NPCID.TravellingMerchant:
                     shop.item[nextSlot].SetDefaults(ModContent.ItemType<Battery>());
-                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(platinum: 0, gold: 0, silver: 10, copper: 0);
+                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(0, 0, 10, 0);
                     nextSlot++;
                     break;
                 case NPCID.Demolitionist:
                     shop.item[nextSlot].SetDefaults(ModContent.ItemType<Battery>());
-                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(platinum: 0, gold: 0, silver: 10, copper: 0);
+                    shop.item[nextSlot].shopCustomPrice = Item.buyPrice(0, 0, 10, 0);
                     nextSlot++;
                     break;
             }
         }
+
         public override void EditSpawnPool(IDictionary<int, float> pool, NPCSpawnInfo spawnInfo)
         {
-            Player player = Main.player[Main.myPlayer];
-            if (ModContent.GetInstance<SpookyTerraria>().beatGame || Main.worldName == SpookyTerrariaUtils.slenderWorldName)
-            {
+            var player = Main.player[Main.myPlayer];
+            if (ModContent.GetInstance<SpookyTerraria>().beatGame ||
+                Main.worldName == SpookyTerrariaUtils.slenderWorldName)
                 pool.Clear(); // No more enemy spawns!.. Because you beat the game, numnut
-            }
-            if (!ModContent.GetInstance<SpookyConfigServer>().excludeAllNPCsBesideCreepyTerrariaOnesFromSpawnPool)
-            {
-                return;
-            }
+
+            if (!ModContent.GetInstance<SpookyConfigServer>().excludeAllNPCsBesideCreepyTerrariaOnesFromSpawnPool) return;
```

Comment:
These oneliners you changed my code to are quite disgusting

Reply:
I mean, you can change the syntax style back... But the || if statements are extremely disgusting and have to be replaced with something more efficient